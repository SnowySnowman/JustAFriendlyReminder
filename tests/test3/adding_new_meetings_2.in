2
beach day with Star and Marco
deMON LANguaGE
06/09/23
wrONGTime
09:00
y
0005
0001
6
