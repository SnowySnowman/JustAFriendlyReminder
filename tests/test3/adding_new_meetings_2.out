Just a friendly reminder! You have these tasks to finish today.
- Protect the Underworld from Intruders.

These tasks need to be finished in the next three days!

You have the following meetings today!
- Adventure with Dad: Portal Hunting! at 09:15
- Celebrating Tom's Birthday in the Underworld. at 18:00

You have the following meetings scheduled over the next week!
- Blood Moon Ball with Star... on 10/09/23 at 20:00
- Castle Hangout with Star and Friends. on 10/09/23 at 17:00
- Fire Magic Training with Mom. on 10/09/23 at 15:30

What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
Please enter a meeting description:
Please enter a date:
Not in valid format. Should be dd/mm/yyPlease enter a date:
Please enter a time:
Not in valid format. Should be hh:mmPlease enter a time:
Ok, I have added beach day with Star and Marco on 06/09/23 at 09:00.
Would you like to share this meeting? [y/n]: Who would you like to share with?
0001 Jackie
0002 Marco
0003 Star
invalid user ID. Please select a valid user ID.
Meeting shared.
What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
